holoviews/plotting/bokeh/callbacks.py

```diff
@@ -9,12 +9,13 @@
     old_bokeh = True
```

Comment:
``old_bokeh`` isn't a great name as one day the latest version will become old! I would consider stating the version explicitly...i.e ``bokeh_XX``.